sidetree/lib/core/MongoDbDidTypeStore.ts

```diff
@@ -0,0 +1,35 @@
+import DidTypeModel from './models/DidTypeModel';
+import IDidTypeStore from './interfaces/IDidTypeStore';
+import MongoDbStore from '../common/MongoDbStore';
+
+/**
+ * Implementation of DidTypeStore that stores the didtype data in
+ * a MongoDB database.
+ */
+export default class MongoDbDidTypeStore extends MongoDbStore implements IDidTypeStore {
+  /** MongoDB collection name under the database where the operations are stored. */
+  public static readonly collectionName: string = 'didtype';
+
+  constructor (serverUrl: string, databaseName: string) {
+    super(serverUrl, MongoDbDidTypeStore.collectionName, databaseName);
+  }
+
+  public async createIndex () {
+    await this.collection.createIndex({ didType: 1 });
```

Comment:
what does the value `1` mean here?

Reply:
The value 1 means the index will be created in ascending order for the didType field.